pkg/server/handler/handlers.go

```diff
@@ -9,11 +9,6 @@ import (
 
 func k8sAPI(sf schema.Factory, apiOp *types.APIRequest) {
 	vars := mux.Vars(apiOp.Request)
-	group := vars["group"]
-	if group == "core" {
-		group = ""
-	}
```

Comment:
Seems like a significant functionality change? Did this fail the linter?

Reply:
It did. Group actually isn't used anywhere - we get it, override it, and then never use it.

Reply:
I'm open to being wrong and having missed something - if you see somewhere it is being used LMK and I can revert.

Reply:
You're right, I missed that it wasn't being used. It was dropped a while ago.